website/content/docs/sync/index.mdx

```diff
@@ -127,6 +127,137 @@ Granularity can be updated. The new granularity only affects secrets newly assoc
+there to prevent users from maliciously or unintentionally leverage elevated permissions on an external system to access
+secrets they normally wouldn't be able to.
+
+Let's assume we have a secret located at `path/to/data/secret1` and a user with write access to the sync feature,
+but no read access to that secret. This scenario is equivalent to this ACL policy:
+
+  <CodeBlockConfig hideClipboard>
+
+    # Allow full access to the sync feature
+    path "sys/sync/*" {
+      capabilities = ["read", "list", "create", "update", "delete"]
+    }
+
+    # Allow read access to the secret mount path/to
+    path "path/to/*" {
+      capabilities = ["read"]
+    }
+
+    # Deny access to a specific secret
+    path "path/to/data/my-secret-1" {
+      capabilities = ["deny"]
+    }
+
+  </CodeBlockConfig>
+
+If a client with this policy tries to read this secret they will receive an unauthorized error:
+
+  <CodeBlockConfig hideClipboard>
+
+    $ vault kv get -mount=path/to my-secret-1
+
+    Error reading path/to/data/my-secret-1: Error making API request.
+
+    URL: GET http://127.0.0.1:8200/v1/path/to/data/my-secret-1
+    Code: 403. Errors:
+
+    * 1 error occurred:
+      * permission denied
+
+  </CodeBlockConfig>
+
+Likewise, if the client tries to sync this secret to any destination they will receive a similar unauthorized error:
+
+  <CodeBlockConfig hideClipboard>
+
+    $ vault write sys/sync/destinations/$TYPE/$NAME/associations/set \
+    mount="path/to" \
+    secret_name="my-secret-1"
+
+    Error writing data to sys/sync/destinations/$TYPE/$NAME/associations/set: Error making API request.
+
+    URL: PUT http://127.0.0.1:8200/v1/sys/sync/destinations/$TYPE/$NAME/associations/set
+    Code: 403. Errors:
+
+    * permission denied to read the content of the secret my-secret-1 in mount path/to
+
+  </CodeBlockConfig>
+
+This read access verification is only done when creating or updating an association. Once the association is created, revoking
+read access to the policy that was used to sync the secret has no effect.
+
+### Collisions and overwrites
+
+Secrets Sync operates with a last-write-wins strategy. If a secret with the same name already exists at the destination,
+Vault overwrites it when syncing a secret. There are also no automatic mechanisms to prevent a principal with sufficient
+privileges at the destination from overwriting a secret synced by Vault.
+
+To prevent Vault from accidentally overwriting existing secrets, it is recommended to use either the name pattern of
+built-in tag as an extra policy condition on the role used to configure a Vault sync destination. For example, the
+following AWS IAM policy prevents Vault from being able to modify secrets that were not created by a sync operation:
+
+  <CodeBlockConfig hideClipboard>
+
+    {
+      "Version": "2012-10-17",
+      "Statement": [
+        {
+          "Effect": "Allow",
+          "Action": [
+            "secretsmanager:*",
```

Comment:
Will this still allow the creation of new secrets?

Reply:
Yes but only if they are created with the expected tag.